testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -675,4 +675,29 @@ public void testOutputColumnsForUpdatingSingleColumn()
         assertThat(event.getIoMetadata().getOutput().get().getColumns().get())
                 .containsExactly("test_varchar");
     }
+
+    @Test
+    public void testCreateTable()
+            throws Exception
+    {
+        runQueryAndWaitForEvents("CREATE TABLE mock.default.create_simple_table (test_column BIGINT)", 2);
+        QueryCompletedEvent event = getOnlyElement(generatedEvents.getQueryCompletedEvents());
+        assertThat(event.getIoMetadata().getOutput().get().getCatalogName()).isEqualTo("mock");
+        assertThat(event.getIoMetadata().getOutput().get().getSchema()).isEqualTo("default");
+        assertThat(event.getIoMetadata().getOutput().get().getTable()).isEqualTo("create_simple_table");
+        assertThat(event.getIoMetadata().getOutput().get().getColumns().get()).containsExactly("test_column");
+    }
+
+    @Test
+    public void testCreateTableLike()
+            throws Exception
+    {
+        runQueryAndWaitForEvents("CREATE TABLE mock.default.create_simple_table (test_column BIGINT, LIKE mock.default.test_table)", 2);
+        QueryCompletedEvent event = getOnlyElement(generatedEvents.getQueryCompletedEvents());
+        assertThat(event.getIoMetadata().getOutput().get().getCatalogName()).isEqualTo("mock");
+        assertThat(event.getIoMetadata().getOutput().get().getSchema()).isEqualTo("default");
+        assertThat(event.getIoMetadata().getOutput().get().getTable()).isEqualTo("create_simple_table");
+        assertThat(event.getIoMetadata().getOutput().get().getColumns().get())
+                .containsExactly("test_column", "test_varchar", "test_bigint");
+    }
```

Comment:
I would like to see `CREATE TABLE AS SELECT` test as well

Reply:
I guess this is already handled here - https://github.com/trinodb/trino/blob/ebeab89e04bc6d55eb0c16a3a7dec85ec16249cd/testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java#L620

Reply:
It is indeed